examples/federated-learning-mistnet-client.Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM tensorflow/tensorflow:1.15.4
```

Comment:
In "mistnet", tf1.15 does not seem to be used. It's better to use the image of Python-slim or Pytorch.

Reply:
Start from ```Python-slim``` needs to install extra libs, which may lead to larger image size.